content/zh-cn/howitworks.md

```diff
@@ -0,0 +1,38 @@
+---
+title: 入门
+slug: getting-started
+top_graphic: 3
+aliases : [/howitworks]
```

Comment:
The "aliases:" field in the Hugo header should not be copied. This leads to inconsistent redirects.

Reply:
Fixed